PokeDemo/CustomViews/ViewBaseClasses/BaseTableViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  BaseTableViewCell.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import UIKit
+
+class BaseTableViewCell: UITableViewCell {
```

Comment:
Why this is required?

Reply:
This is made to write common code for all table view cells.

Reply:
It is recommended to use protocol instead of inheritance. Refer this - https://medium.com/bleeding-edge/nicer-reuse-identifiers-with-protocols-in-swift-97d18de1b2df

Reply:
Covered in 32fb355207022acc0d4f94754537ed669f817815